### PR TITLE
11.0 mig stock orderpoint automatic creation

### PR DESCRIPTION
**Pull request by andreasdsp**

### CODE REVIEW

**Comment by ghost** [comment]
How could i support this (and others) modules to get merged?

**Comment by pedrobaeza** [comment]
Review them: https://odoo-community.org/page/review

**Comment by ghost** [comment]
Sorry. i asking often same, but i want learn it :) i cant start the runbot instance, i dont understood the system . But i test the module on my own testing-instance and it seems ok.

**Review by HaraldPanten — APPROVED** [review]
Functional review

:+1:

File: stock_orderpoint_automatic_creation/README.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+   data.
+#. Also you can configure orderpoint creation in product and product_category
+   checking 'Yes' or 'No' in the proper field.
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/153/10.0
```
Comment (LoisRForgeFlow) [inline stock_orderpoint_automatic_creation/README.rst:28]: 11.0

File: stock_orderpoint_automatic_creation/__init__.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
```
Comment (LoisRForgeFlow) [inline stock_orderpoint_automatic_creation/__init__.py:1]: this line [is not needed anymore in python 3](https://stackoverflow.com/questions/37048761/coding-utf-8-on-python3/37048998#37048998)

File: stock_orderpoint_automatic_creation/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
```
Comment (LoisRForgeFlow) [inline stock_orderpoint_automatic_creation/__manifest__.py:1]: same for all files

File: stock_orderpoint_automatic_creation/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Stock Orderpoint Automatic Creation",
+    "version": "11.0.1.0.0",
+    "author": "AvanzOSC, "
+              "Tecnativa, "
+              "Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "website": "http://www.odoomrp.com",
```
Comment (LoisRForgeFlow) [inline stock_orderpoint_automatic_creation/__manifest__.py:13]: https://github.com/OCA/stock-logistics-warehouse

File: stock_orderpoint_automatic_creation/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    "depends": [
+        "stock",
+    ],
+    "category": "Inventory, Logistic, Storage",
```
Comment (LoisRForgeFlow) [inline stock_orderpoint_automatic_creation/__manifest__.py:17]: Just `Inventory` is better

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]
Some styling/standars comments. Other than that code looks good.

**Comment by HaraldPanten** [comment]
Hi @andreasdsp ,

Any news about this PR?

Thanks!

**Comment by andreasdsp** [comment]
Hi, 
I'm sorry, i can't continue this because i'm still busy on another project now.
Thanks.

**Comment by rousseldenis** [comment]
Supersedes by #502

**Pull request closed without merging**